validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -335,13 +333,13 @@ private void createBlockIfNecessary(BeaconStateWithCache state, BeaconBlock oldB
         BLSSignature epochSignature = epochSignatureTask.get();
         newBlock.getBody().setRandao_reveal(epochSignature);
         List<ProposerSlashing> slashingsInBlock = newBlock.getBody().getProposer_slashings();
-        slashings.forEach(
-            slashing -> {
-              if (!state.getValidators().get(slashing.getProposer_index().intValue()).isSlashed()) {
-                slashingsInBlock.add(slashing);
-              }
-            });
-        slashings = new LinkedBlockingQueue<>();
```

Comment:
This variable is actually used in the commented out code. The one which detects and submits proposer slashings.

Reply:
The `slashings` variable still exists and is kept up to date - I've just changed it to use a more typical approach of draining the queue instead of iterating then replacing. That happens to also make error prone happy since before it was read but never updated (due to the insert calls being commented out).